src/components/TaskRunnerPane/TaskRunnerPane.js

```diff
@@ -5,7 +5,11 @@ import { connect } from 'react-redux';
 import * as actions from '../../actions';
 import { GUPPY_REPO_URL } from '../../constants';
 import { getSelectedProjectId } from '../../reducers/projects.reducer';
```

Comment:
Nit: unused `getSelectedProjectId` (not part of your changes but slipped in)

Reply:
Good catch! I wound up actually using it, so it's no longer a warning :)